css/amp.css

```diff
@@ -49,12 +49,13 @@ body {
 
 
 /**
- * iOS-Embed Wrapper mode. The `body` is wrapper into `i-amp-html-wrapper`
+ * iOS-Embed Wrapper mode. The `body` is wrapped into `i-amp-html-wrapper`
  * element.
  */
 html.-amp-ios-embed {
   overflow-y: auto !important;
   -webkit-overflow-scrolling: touch !important;
+  position: static;
```

Comment:
It only currently causes issues on iOS and this will be launched soon. I'll be open to extend it to other viewport types post-launch as part of the cleanup process.